plugins/outputs/graphite/graphite.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 type Graphite struct {
-	GraphiteTagSupport bool   `toml:"graphite_tag_support"`
-	GraphiteSeparator  string `toml:"graphite_separator"`
+	GraphiteTagSupport     bool   `toml:"graphite_tag_support"`
+	GraphiteTagNewSanitize bool   `toml:"graphite_tag_new_sanitize"`
```

Comment:
Instead of a bool option, how about a string option of sanitize which defaults to "strict" (the previous behavior) just allows for more future extension

Reply:
Yeah, that makes sense. Any thoughts on the name for the mode at the heart of this PR? Maybe "compatible", or "full"?

Reply:
I think "compatible" makes sense as it's in line with the graphite spec